judgels-backends/jophiel/jophiel-app/src/main/java/judgels/jophiel/user/UserStore.java

```diff
@@ -123,6 +123,13 @@ public void updateUserPassword(String userJid, String newPassword) {
                 .collect(Collectors.toMap(User::getUsername, p -> p));
     }
 
+    public Map<String, String> translateUsernamesToJids(Set<String> usernames) {
```

Comment:
So actually, the previous method (`getUsersByUsernames()`) should now be deleted. There is one more call to the method (in UserRatingResource), which can be replaced by this new method.